monkey/infection_monkey/payload/ransomware/ransomware_options.py

```diff
@@ -9,6 +10,11 @@
 logger = logging.getLogger(__name__)
 
 
+class EncryptionAlgorithm(StrEnum):
+    BIT_FLIP = "bit_flip"
```

Comment:
Should this string value be all caps, since it's all caps in the schema, and we use upper() below? Or does it also compare with the variable name?

Reply:
I don't know that it matters much. It will matter even less when the ransomware config gets converted to using pydantic.